src/cargo/core/package.rs

```diff
@@ -24,25 +25,26 @@ pub struct Package {
 
 #[derive(RustcEncodable)]
 struct SerializedPackage<'a> {
```

Comment:
will name `package_id` just `id` for simplicity and `source_id` just `source` for consistency with `SerializedDependency` .

So the `source` will be included in both a `Package` and a `Dependency`.